src/engine.py

```diff
@@ -0,0 +1,9 @@
+from jinja2 import Environment, FileSystemLoader
+from dateutil import parser
+
+engine = Environment(loader=FileSystemLoader(["templates", "automation/templates"]))
+
+def datetime_format(value, format="%d-%m-%y"):
+    return parser.isoparse(value).strftime(format)
+
+engine.filters["datetime_format"] = datetime_format
```

Comment:
The lack of a newline here seems like the pre-commit hooks weren't run against this file.